DeadCrumbs/app/src/main/java/dead/crumbs/client/apis/DeadCrumbsApi.kt

```diff
@@ -0,0 +1,202 @@
+/**
+ * DeadCrumbs API
+ * This is the API for for the DeadCrumbs application.
+ *
+ * OpenAPI spec version: 1
+ * 
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+package dead.crumbs.client.apis
+
+import io.swagger.client.models.Location
+import io.swagger.client.models.User
+
+import io.swagger.client.infrastructure.*
+
+class DeadCrumbsApi(basePath: kotlin.String = "/") : ApiClient(basePath) {
+
+    /**
+     * returns the location associated with username
+     * 
+     * @param username The username of the user. 
+     * @return Location
+     */
+    @Suppress("UNCHECKED_CAST")
+    fun getLocation(username: kotlin.String): Location {
+        
+        val localVariableConfig = RequestConfig(
+                RequestMethod.GET,
+                "/Location/{username}".replace("{" + "username" + "}", "$username")
+        )
+        val response = request<Location>(
+                localVariableConfig
+        )
+
+        return when (response.responseType) {
+            ResponseType.Success -> (response as Success<*>).data as Location
+            ResponseType.Informational -> TODO()
+            ResponseType.Redirection -> TODO()
+            ResponseType.ClientError -> throw ClientException((response as ClientError<*>).body as? String ?: "Client error")
+            ResponseType.ServerError -> throw ServerException((response as ServerError<*>).message ?: "Server error")
+        }
+    }
+    /**
+     * return the user
+     * 
+     * @param username The username of the user. 
+     * @return User
+     */
+    @Suppress("UNCHECKED_CAST")
+    fun getUser(username: kotlin.String): User {
+        
+        val localVariableConfig = RequestConfig(
+                RequestMethod.GET,
+                "/User/{username}".replace("{" + "username" + "}", "$username")
```

Comment:
Why write "{" + "username" + "}", when you could just write "{username}"?

Reply:
This is autogenerated